Classes/TBODeveloperOverlayKVDebugger/TBODeveloperOverlayKVDebugger.m

```diff
@@ -96,4 +106,17 @@ - (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInte
     return [self.datasource titleForSection:section];
 }
 
+#pragma mark helper
+
+- (Class)detailViewControllerClassForValue:(id)value {
```

Comment:
I was wrong here. If we only use the valueClass we couldn't distinct between Bool-NSNumers and Number-NSNumbers.